src/pudl/metadata/resources/eia860.py

```diff
@@ -571,6 +571,54 @@
         "sources": ["eia860"],
         "etl_group": "eia860",
     },
+    "boiler_cooler_assn_eia860": {
+        "description": "A table that links EIA boiler ID to EIA cooling system ID.",
```

Comment:
Same here and throughout, think this should be "IDs". Otherwise descriptions look clear.